app/assets/javascripts/models/page/access_page.js.coffee

```diff
@@ -30,6 +30,7 @@ ns.AccessPage = do (ko,
                     ) ->
 
   class AccessPage
+
```

Comment:
Didn't like this being that close? ;)

Reply:
Sadly, I noticed this one right after I committed. Happy to bring it back to its original home.